README.md

```diff
@@ -474,14 +474,15 @@ To capture the entire viewport, pass `document` as the target element to the ass
 await expect(document).to.be.golden();
 ```
 
-#### Rendering Fullscreen Fixtures
+#### Turning Off Full Page Padding
 
-To render a fullscreen fixture that fills the viewport, pass a `fullscreen` option to `fixture()`:
+By default, the `body` of the page has `38px` of padding. For elements meant to take up the entire page with no bordering whitespace, you may want to turn that off:
```

Comment:
Just trying to avoid you/your/etc.:

```suggestion
By default, the the page hosting vdiff fixtures has `38px` of padding. For fixtures which are meant to fill the entire page with no surrounding whitespace, the page padding can be disabled:
```

Reply:
Awesome, much better